packages/dashboard/src/locales/ko-KR.json

```diff
@@ -312,6 +312,8 @@
     "labs_settings_market_trend_source": "디폴트 추세 소스",
     "labs_settings_swap": "스왑 설정",
     "labs_settings_swap_network": "{{network}} 네트워크 디폴트 추세 소스",
+    "labs_cryptoartai": "CRYPTOART.AI",
+    "labs_cryptoartai_desc": "Connect CryptoArt Digital Artworks. The Internationale is a classic anthem that has been a shared memory for generations.",
```

Comment:
Please leave other languages empty if you don't know how to translate them.

Reply:
Thanks,  @Jack-Works - Fixed.